src/main.py

```diff
@@ -55,6 +55,7 @@ class Options:
     abort_exceptions: bool = False
     better_only: bool = False
     best_only: bool = False
+    score_threshold: int = 1
```

Comment:
this is an Optional[int], and the default should probably be None

Reply:
thought I had it as optional int,  will update